sunpy/util/tests/test_sysinfo.py

```diff
@@ -16,11 +16,29 @@ def test_find_dependencies():
 
     missing, installed = find_dependencies(package="sunpy", extras=["required", "all"])
     assert missing == {}
-    assert sorted(list(installed.keys())) == sorted(['astropy', 'numpy', 'parfive', 'packaging', 'asdf',
-                                                     'dask', 'sqlalchemy', 'scikit-image', 'scipy', 'glymur',
-                                                     'matplotlib', 'mpl-animators', 'reproject',
-                                                     'beautifulsoup4', 'drms', 'python-dateutil', 'tqdm',
-                                                     'zeep', 'cdflib', 'h5netcdf', 'h5py', 'pandas'])
+    assert sorted(list(installed.keys())) == sorted(['asdf',
```

Comment:
Why do this?

Reply:
Viewing the diff was a pain, this makes it a lot easier.